modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java

```diff
@@ -921,6 +958,16 @@ else if (Objects.equals(
 					}
 				}
 
+				for (int i = 0; i < assigneeGroupIds.size(); i++) {
+					roleIds = mapGroupRoleIds.get(assigneeGroupIds.get(i));
+
+					if (roleIds.contains(assigneeIds.get(i))) {
+						reviewer = true;
+
+						break;
+					}
+				}
+
```

Comment:
Hello @rafaprax, we can create a new method with this changes to facilitate other developer understanding?, for example, `isReviewer`

Reply:
Yeap. We can.